content/03.categorize.md

```diff
@@ -106,18 +106,17 @@ They trained a deep neural network on approximately 100,000 images extracted fro
-Specifically, they first extracted all diseases mentioned in the reports using a state-of-the-art NLP tool, then applied a newly-developed negation and uncertainty detection tool (NegBio) to filter negative and equivocal findings in the reports.
-Evaluation on three independent datasets demonstrated that NegBio is highly accurate for detecting negative and equivocal findings (~90% in F-measure, which balances precision and recall [@doi:10.1038/nmeth.3945]).
-These highly-accurate results meet the need to generate a corpus with weak labels, which serves as a solid foundation for the later process of image classification.
-The resulting dataset [@url:https://nihcc.app.box.com/v/ChestXray-NIHCC] consists of 112,120 frontal-view chest X-ray images from 30,805 patients, and each image is associated with one or more weakly-labeled pathology category (e.g. pneumonia and cardiomegaly) or "normal" otherwise.
-Further, Wang et al.
-[@arxiv:1705.02315] used this dataset with a unified weakly-supervised multi-label image classification framework, to detect common thoracic diseases.
+Although rich clinical information is stored in EHRs, manually annotating them is not only costly and time consuming but also requires domain knowledge.
+Take chest X-ray studies for example, a radiologist usually spends a few minutes per x-ray for the whole process. 
+As a result, providing deep learning algorithms for chest X-ray image analysis with large-scale human-labeled training data is highly challenging. 
+Alternatively, researchers have investigated how to text-mine annotations from the corresponding narrative portions of the EHR as the silver standard benchmark dataset [@doi:10.3115/1572392.1572411].
+However, the text of clinical notes is also difficult to interpret because it is often in the form of unstructured free text. 
+To this end, Wang et al. [@arxiv:1705.02315] proposed to build predictive DL models through the use of images with *weak labels* (automatically generated data not verified by human and may be noisy or incomplete).
+In this case, they applied a series of natural language processing (NLP) techniques to the associated chest X-ray radiological reports. 
+Specifically, they first extracted all diseases mentioned in the reports using a state-of-the-art NLP tool, then applied a newly-developed negation and uncertainty detection tool (NegBio [@arxiv:1712.05898]) to filter negative and equivocal findings in the reports. 
+Evaluation on four independent datasets demonstrated that NegBio is highly accurate for detecting negative and equivocal findings (~90% in F-measure, which balances precision and recall [@doi:10.1038/nmeth.3945]).
+The resulting dataset [@url:https://nihcc.app.box.com/v/ChestXray-NIHCC] consists of 112,120 frontal-view chest X-ray images from 30,805 patients, and each image is associated with one or more *text-mined* (weakly-labeled) pathology category (e.g. pneumonia and cardiomegaly) or "no finding" otherwise.
```

Comment:
Minor, but I think "category" here should be "categories" because it's `one or more...`